build.sh

```diff
@@ -77,8 +77,8 @@ function build_wikibase {
         --build-arg WIKIBASEEDTF_COMMIT="$WIKIBASEEDTF_COMMIT" \
         --build-arg WIKIBASELOCALMEDIA_COMMIT="$WIKIBASELOCALMEDIA_COMMIT" \
         \
-        --build-arg MW_WG_SITENAME="$MW_WG_SITENAME" \
-        --build-arg MW_WG_LANGUAGE_CODE="$MW_WG_LANGUAGE_CODE" \
+        --build-arg MW_WG_SITENAME="wikibase" \
+        --build-arg MW_WG_LANGUAGE_CODE="en" \
```

Comment:
Maybe just remove this as it defaults in Dockerfile

Reply:
Same for all of the below similar entries. Maybe we come back to this in a PR that keeps all of the default like this just in the Dockefiles.

Reply:
https://phabricator.wikimedia.org/T368145